datadog_checks_base/datadog_checks/base/utils/db/query.py

```diff
@@ -240,6 +255,7 @@ def compile(
         self.extra_transformers = tuple(extra_data)
         self.base_tags = tags
         self.collection_interval = collection_interval
+        self.metric_name_raw = metric_prefix is not None
```

Comment:
why do we need this boolean? I don't see it really being used

Reply:
It's been used https://github.com/DataDog/integrations-core/blob/2e91e6db7c3b3196c7109024f4245894f5150fcf/datadog_checks_base/datadog_checks/base/utils/db/core.py#L131 to ensure `raw` is set to true to skip prepending the integration namespace

Reply:
ah OK, my bad. sounds good 👍